src/pages/timecard/Timecard.spec.jsx

```diff
@@ -82,9 +82,32 @@ describe('Timecard', () => {
 
     expect(screen.queryByText('Add a new time period')).toBeFalsy();
     expect(screen.queryByText('Shift')).toBeFalsy();
+    expect(screen.queryByText('Non-working day')).toBeFalsy();
     expect(screen.getByText('Scheduled rest day')).toBeTruthy();
   });
 
+  it('should render the NonWorkingDay component when time period type is NWD', async () => {
```

Comment:
The test name needs to change to refer to the SimpleTimePeriod component. Same goes for the SRD one on the test above.
```suggestion
  it('should render the SimpleTimePeriod component when time period type is NWD', async () => {
```

Reply:
I've changed this from being an individual test to a test.each, so the title has changed now

Reply:
perfect.